src/invariant/LedgerEntryIsValid.cpp

```diff
@@ -301,10 +301,6 @@ LedgerEntryIsValid::checkIsValid(ClaimableBalanceEntry const& cbe,
     {
         return "ClaimableBalance amount is not positive";
     }
-    if (cbe.reserve <= 0)
```

Comment:
should there be a check that a `ClaimableBalanceEntry` is always sponsored? (`LedgerEntryIsValid` invariant)

Reply:
Good point. Added a check.